src/tools/generateImage.ts

```
@@ -83,6 +83,20 @@
   return generateImageCommon(context, prompt, false);
 };
 
+export function createUploadedImageResult(
+  imageData: string,
+  fileName: string,
+  prompt: string,
+): ToolResult {
+  return {
+    toolName: toolName,
```

ESLint

[error] Expected property shorthand.

(object-shorthand)